src/sentry/api/endpoints/organization_user_rollback.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import status
+from rest_framework.permissions import BasePermission
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers.base import serialize
+from sentry.api.serializers.models.userrollback import UserRollbackSerializer
+from sentry.constants import ROLLBACK_ENABLED_DEFAULT
+from sentry.models.organization import Organization
+from sentry.models.rollbackorganization import RollbackOrganization
+from sentry.models.rollbackuser import RollbackUser
+
+
+class MemberPermission(BasePermission):
+    scope_map = {
+        "GET": ["member:read"],
+    }
+
+
+@region_silo_endpoint
+class OrganizationRollbackUserEndpoint(OrganizationEndpoint):
```

Comment:
why not use `OrganizationMemberEndpoint`?

Reply:
Good call, though I think the member_id that that class provides would be less efficient - the user identifier in the database is the user_id, not member_id. Feel free to correct me if I'm misunderstanding